source/Nevermore/TableColumnNameResolvers/JsonLastTableColumnNameResolver.cs

```diff
@@ -14,10 +14,6 @@ public JsonLastTableColumnNameResolver(IRelationalStore store)
 
         public string[] GetColumnNames(string schemaName, string tableName)
         {
-            var schemaClause = string.IsNullOrEmpty(schemaName)
-                ? ""
-                : "AND s.name = @schemaName";
-
```

Comment:
I checked all usages and schema shouldn't be null anymore (and if it's empty it's a configuration error).